robocop/checkers/naming.py

```diff
@@ -574,8 +580,15 @@ def visit_Setup(self, node):  # noqa
 
     def visit_LibraryImport(self, node):  # noqa
         self.check_setting_name(node.data_tokens[0].value, node)
-        with_name = node.get_token(*self.ALIAS_TOKENS)
-        if with_name is None:
+        if ROBOT_VERSION.major < 6:
+            arg_nodes = node.get_tokens(Token.ARGUMENT)
+            # ignore cases where 'AS' is used to provide library alias for RF < 5
```

Comment:
I think AS was introduced in RF 6.0, so the code for < RF 6 shouldn't care if there is AS or not?

Reply:
If you open `tests/atest/rules/naming/empty_library_alias/test.robot` file in your IDE with RF 5 installed, you'll see that it reports this problem on the lines where `AS` is used, while IMO it should ignore such cases, instead of reporting an issue there.

It detects `AS` and following tokens as arguments.

See example here (RF 5 installed):

<img width="850" alt="image" src="https://user-images.githubusercontent.com/7412964/226833672-cd3fcffd-d243-487f-b714-b6f44ac3814a.png">